extension/src/popup/views/FullscreenSuccessMessage/index.tsx

```diff
@@ -26,38 +50,16 @@ const MnemonicPhraseConfirmedMessage = () => (
         responsibility.
       </p>
     </div>
-    <div className="FullscreenSuccessMessage__infoBlock">
-      <InfoBlock variant={InfoBlock.variant.warning}>
-        <div className="InfoBlock__content">
-          <div className="InfoBlock__header">
-            Avoid scams and keep your account safe:
-          </div>
-          <ul>
-            <li>
-              Freighter will never ask for your recovery phrase unless you're
-              actively importing your account using the browser extension -
-              never on an external website.
-            </li>
-            <li>
-              Always check the domain of websites you’re using Freighter with
-            </li>
-            <li>
-              Freighter cannot recover your account if you lose your recovery
-              phrase, so keep it safe
-            </li>
-          </ul>
-        </div>
-      </InfoBlock>
-    </div>
+    <AvoidScamsWarningBlock />
     <SubmitButtonWrapper>
       <Button
         fullWidth
         onClick={() => {
           emitMetric(METRIC_NAMES.accountCreatorFinished);
-          window.close();
+          navigateTo(ROUTES.pinExtension);
```

Comment:
hmm is it worth only nav'ing to ROUTES.pinExtension in Chrome since this doesn't appear to apply on Firefox

Sorry! I should have caught this beforehand

Reply:
nice catch updated 👍  using the old images / copy if it isn't chrome